DefaultTabBar.js

```diff
@@ -20,6 +20,8 @@ const DefaultTabBar = React.createClass({
     inactiveTextColor: React.PropTypes.string,
     textStyle: Text.propTypes.style,
     tabStyle: View.propTypes.style,
+    underlineStyle: View.propTypes.style,
+    tabBarPosition: React.PropTypes.oneOf(['top', 'bottom', 'overlayTop', 'overlayBottom', ]),
```

Comment:
this code: 
```javascript
const tabUnderlineStyle = {
    position: 'absolute',
    width: containerWidth / numberOfTabs,
    height: this.props.underlineHeight,
    backgroundColor: this.props.underlineColor,
    bottom: 0,
};
```
when` tabBarPosition="bottom"`, I hope `top: 0`, `bottom` value  can not be changed by `underlineStyle` to 'auto' like css( bottom: 'auto'), it must be a number